Pod/Classes/URBNArrayDataSourceAdapter.m

```diff
@@ -93,6 +93,27 @@ - (void)appendItems:(NSArray *)newItems inSection:(NSInteger)section {
     }
 }
 
+- (void)appendSectionWithItems:(NSArray *)newItems {
+    
+    if ([self isSectioned]) {
+        void (^updateData)() = ^{
+            [self.items addObject:newItems];
+        };
+        
+        if (self.tableView) {
+            updateData();
+            [self.tableView insertSections:[NSIndexSet indexSetWithIndex:(self.items.count - 1)] withRowAnimation:UITableViewRowAnimationFade];
```

Comment:
I actually tried `self.rowAnimation` and it caused the insertion to be pushed from the left which was ugly. In this case, would I change the datasource’s rowAnimation to be fade, perform the operation, then change it back?